test/examples/0-showcase.jsx

```diff
@@ -84,18 +88,23 @@ class ShowcaseLayout extends React.Component {
         <button onClick={this.onCompactTypeChange}>
           Change Compaction Type
         </button>
+        <div className="droppable-element" draggable={true} unselectable="on">
```

Comment:
This is cool, it will be even better if this awesome feature has its own separate page for a demo I think.

Reply:
@n1ghtmare There are no links between documentation pages, so if I move this example into separate page, users won't be able to see it. I think we should update documentation system. I can replace current docs with https://docusaurus.io/ (or something like that)

What do you think?

Reply:
Sure, in the long run, having a better docs page would be great, I fully agree. However for now, why not just add it to the demo selection in the README as number 16 (or whatever the next number was) and have it be on its own (same like the rest, with "view source" and all).

Reply:
@n1ghtmare done